Consul.Test/AgentTest.cs

```diff
@@ -1025,6 +1025,15 @@ public async Task Agent_Metrics()
             Assert.NotNull(agentMetrics.Response.Samples);
         }
 
+        [Fact]
+        public async Task Agent_CARoots()
+        {
+            var caRoots = await _client.Agent.GetCARoots();
+            Assert.NotEqual((ulong)0, caRoots.LastIndex);
+            Assert.Single(caRoots.Response.Roots);
+            Assert.Equal("11111111-2222-3333-4444-555555555555.consul", caRoots.Response.TrustDomain);
```

Comment:
This test could test all fields in the response (at least for fields being non-null). That would assure us that all fields in the newly added structures are named correctly.

Reply:
Just added the test assertions, ready for further review!